packages/terra-i18n-plugin/src/I18nAggregatorPlugin.js

```diff
@@ -70,8 +71,13 @@ function aggregateTranslationMessages(options, inputFileSystem) {
   let languageMessages = {};
   supportedLocales.forEach((language) => { languageMessages[language] = {}; });
 
-  // Aggregate translation messages for the directory
-  languageMessages = aggregateDirectory(languageMessages, options.baseDirectory, inputFileSystem);
+  // Aggregate translation messages for node_modules
```

Comment:
Nit: I know its a bit cleaner to just pass true & false to aggregrateDirectory, but it might improve readability to add a variable `searchNodeModules` so it indicates why true & false are passed

Reply:
+1